fluent-plugin-events/test/plugin/test_in_events.rb

```diff
@@ -186,4 +186,65 @@ def connect_kubernetes_with_api_version(driver)
     
     driver.start_monitor
   end
+
+  sub_test_case 'exception from kubeclient will not interrupt execution of start monitor' do
```

Comment:
`create_config_map` and `initialize_resource_version` are not called within `start_monitor` right?

Reply:
Yes. Need to change this description then.